app/policies/application_policy.rb

```diff
@@ -57,7 +57,32 @@ def initialize(user, scope)
     end
 
     def resolve
+      return scope.all unless user.rbac_enabled?
+
+      graphql_query_by_id? ? graphql_id_query : resolve_scope
+    end
+
+    def resolve_scope
+      raise Exceptions::NotAuthorizedError, "Read access not authorized for #{scope}" unless permission_check('read', scope)
```

Comment:
should this check part of `resolve` method?

Reply:
No, `request` and `action` don't do this check